apps/server/src/models/Product.ts

```diff
@@ -0,0 +1,18 @@
+export class Product {
+    id: number = 0;
+    name: string = '';
+    description?: string = '';
+    image?: string = '';
+    private createdAt?: string = '';
+    updatedAt?: string = '';
+
+    constructor({ name, description, image }: { name: string, description?: string, image?: string }) {
+        this.name = name;
+        this.description = description;
+        this.image = image;
+        this.createdAt = Date.now().toString();
+        this.updatedAt = Date.now().toString();
+    }
+
+
```

Comment:
HINT: If all this class has is a constructor, a Typescript Interface or type could be better option.